docs/cli-guide_5/master.adoc

```diff
@@ -12,6 +12,9 @@ include::topics_5/templates/document-attributes.adoc[]
 
 = {UserCLIBookName}
 
+//Inclusive language statement
+include::topics_5/making-open-source-more-inclusive.adoc[leveloffset=+1]
```

Comment:
The RHEL docs do not specify a [level offset](https://gitlab.cee.redhat.com/red-hat-enterprise-linux-documentation/rhel-8-docs/-/blob/master/enterprise/titles/planning/preparing-for-disaster-recovery-with-identity-management/master.adoc). 

Please remove the `leveloffset=+1` from each `include::` in all assemblies and then I will merge.

Reply:
I looked in their assembly files and they do specify the level offset. It's +1 for all includes.